src/os/bluestore/BlueStore.h

```diff
@@ -71,7 +74,10 @@ class BlueStoreRepairer;
 class SimpleBitmap;
 //#define DEBUG_CACHE
 //#define DEBUG_DEFERRED
-
+#ifdef WITH_CPUTRACE
```

Comment:
I'd combine this block with the above one which includes cputrace. Just to have WITH_CPUTRACE stuff in a single place